dom0/sd-dom0-qvm-rpc.sls

```diff
@@ -76,7 +76,23 @@ dom0-rpc-qubes.USB:
     - marker_start: "### BEGIN securedrop-workstation ###"
     - marker_end: "### END securedrop-workstation ###"
     - content: |
-        $anyvm $tag:sd-workstation deny
+        sd-export-usb sys-usb allow
```

Comment:
these policy changes result in failing RPC tests. Could you please update the test policies in https://github.com/freedomofpress/securedrop-workstation/blob/307-udev-export-devices/tests/vars/qubes-rpc.yml ?

Reply:
Updated.